src/renderer/components/dex/MarketSelection.vue

```diff
@@ -71,16 +73,65 @@ export default {
       return this.storeStateCurrentMarket && this.$store.state.holdings.length ?
         this.$services.neo.getHolding(this.storeStateCurrentMarket.baseAssetId).unitValue : 0;
     },
+
+    close24Hour() {
+      return this.todayTrades.length > 0 ?
+        this.todayTrades[0].price : 0;
+    },
+
+    open24Hour() {
+      return this.todayTrades.length > 0 ?
+        this.todayTrades[this.todayTrades.length - 1].price : 0;
+    },
+
+    low24Hour() {
+      return this.todayTrades.length > 0 ?
+        _.minBy(this.todayTrades, trade => Number(trade.price)).price : 0;
+    },
+
+    high24Hour() {
+      return this.todayTrades.length > 0 ?
+        _.maxBy(this.todayTrades, trade => Number(trade.price)).price : 0;
+    },
+
+    volume24Hour() {
+      return this.todayTrades.length > 0 ?
+        _.sumBy(this.todayTrades, trade => Number(trade.quantity)) : 0;
+    },
+
+    change24HourPercent() {
+      return this.todayTrades.length > 0 ?
+        Math.round(
+          ((this.close24Hour - this.open24Hour) / this.open24Hour) * 10000) / 100 : 0;
+    },
+
+    change24Hour() {
+      return this.close24Hour - this.open24Hour;
+    },
+
```

Comment:
Instead of having the logic of actually calculating these properties, maybe we should just put the tertiary logic here in computed. The reason being is that this isn't the only view that will need these values (change24Hour is most likely used in many views, for instance) and we don't want to have to duplicate this code every time.